tests/network/vmi_passt.go

```diff
@@ -62,8 +62,16 @@ var _ = SIGDescribe("[Serial] Passt", decorators.PasstGate, Serial, func() {
 	Context("VirtualMachineInstance with passt binding mechanism", func() {
 
 		It("should report the IP to the status", func() {
```

Comment:
This is no longer reflecting what the test does.
Perhaps "should apply the interface configuration" is a better fit now.

Reply:
Done